.github/workflows/molecule-tests.yml

```diff
@@ -45,3 +46,13 @@ jobs:
           cd ./linchpin/provision/roles/openstack
           molecule converge -s delegated
           molecule destroy -s delegated
+      - name: "Execute Azure molecule tests"
```

Comment:
These same tests need to be run in `.github/workflows/molecule-tests-centos7.yml`

Reply:
Thanks!